apps/integration/integration/containers/stack.py

```diff
@@ -0,0 +1,39 @@
+from testcontainers.compose import DockerCompose
+import pytest
+from integration import SYCAMORE_ROOT
+from typing import List
+
+
+def docker_compose(services: List[str] = []):
+    """
+    Get a docker compose object that controls some set of the sycamore stack
+    :param services: the list of services to control. [] -> all services
+    :return: the docker compose object
+    """
+    if len(services) > 0:
```

Comment:
I'd just have `if not services:` so people can pass in None, too.  Reads better, too.